src/SFA.DAS.AssessorService.Web.Staff/Controllers/ExcelAwareController.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace SFA.DAS.AssessorService.Web.Staff.Controllers
+{
+    using Microsoft.AspNetCore.Mvc;
+    using System.Collections.Generic;
+    using System.Data;
+    using System.Linq;
+
+    public class ExcelAwareController : Controller
```

Comment:
This is a weird name for a controller.  Is there a reason to call it ExcelAware?

Reply:
I wanted to reuse the excel data table functionality so made a base class for ReportsController and RoatpController. Happy to go with other suggestions on naming